distribution/integration/tests-integration/src/test/java/org/wso2/carbon/integration/tests/carbontools/RunBuildXMLTestCase.java

```diff
@@ -0,0 +1,144 @@
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.integration.tests.carbontools;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.engine.context.AutomationContext;
+import org.wso2.carbon.automation.engine.context.ContextXpathConstants;
+import org.wso2.carbon.automation.test.utils.common.TestConfigurationProvider;
+import org.wso2.carbon.integration.common.exception.CarbonToolsIntegrationTestException;
+import org.wso2.carbon.integration.common.utils.CarbonCommandToolsUtil;
+import org.wso2.carbon.integration.common.utils.CarbonIntegrationBaseTest;
+
+import java.io.File;
+import java.io.IOException;
+
+import static org.testng.Assert.assertTrue;
+
+// This class is to test build.xml by running ant check repository lib folder for jars and
+// run ant localize and check repository/components/dropins for language bundle
+
+public class RunBuildXMLTestCase extends CarbonIntegrationBaseTest {
+
+    private static final Log log = LogFactory.getLog(RunBuildXMLTestCase.class);
+    AutomationContext context;
+    String carbonHome;
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws Exception {
+        super.init();
+        context = new AutomationContext("CARBON", "carbon002",
+                                        ContextXpathConstants.SUPER_TENANT,
+                                        ContextXpathConstants.ADMIN);
+        carbonHome = CarbonCommandToolsUtil.getCarbonHome(context);
+    }
+
+    @Test(groups = {"wso2.as"}, description = "Server restart test")
+    public void testBuildXMLGenerateRemoteRegistryClients()
+            throws CarbonToolsIntegrationTestException {
+        boolean isJarCreated = false;
+        Process process = null;
+        try {
+
+            File folder = new File(carbonHome + File.separator + "repository" + File.separator + "lib");
+            File[] listOfFilesBeforeRunAntCommand = folder.listFiles();
+            String[] cmdArray;
+            if (CarbonCommandToolsUtil.isCurrentOSWindows()) {
+                cmdArray = new String[]{"cmd.exe", "/c", "start", "ant" };
+            } else {
+                cmdArray = new String[]{"ant"};
+            }
+            process = CarbonCommandToolsUtil.runScript(carbonHome + File.separator + "bin", cmdArray);
+            long startTime = System.currentTimeMillis();
+            long timeout = 20000;
+            while ((System.currentTimeMillis() - startTime) < timeout) {
+                File[] listOfFilesAfterRunAntCommand = folder.listFiles();
+                if (listOfFilesBeforeRunAntCommand != null && listOfFilesAfterRunAntCommand != null) {
+                    if (listOfFilesAfterRunAntCommand.length > listOfFilesBeforeRunAntCommand.length) {
+                        log.info("Jars created successfully");
```

Comment:
incorrect log message. Ant script will copy jars for plugins to lib. Not creating..